### PR TITLE
Added a lame greeting 'Foobar'

### PR DESCRIPTION
**Pull request by aaron-weiss**
Added a greeting as part of #1

### CODE REVIEW

**Comment by medo64** [comment]
I approve of Foobar! :)

File: game/src/constants.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const gravity = new BABYLON.Vector2(0, -9.8);
 const flightForce = 5; // The amount of force applied when the "fly" button is pressed
 
 const greetings = [
+    "It's a Foobar!",
```
Comment (medo64) [inline game/src/constants.js:5]: I like it's capitalized!

Comment (sethborne) [inline game/src/constants.js:5]: does this line needs a character break?

Comment (medo64) [reply game/src/constants.js:5]: I believe it should be fine.

**Pull request merged**